### PR TITLE
Refactor run_non_RPC

### PR DESCRIPTION
**Pull request by gasull**
run_non_RPC has too many branches, making it hard to follow.  The
refactoring extracts two new functions: create_wallet and
restore_wallet.  I also did some minor changes in style to fix pylint
errors, like keeping lines under 80 characters.

See too-many-branches (R0912) on
https://pylint.readthedocs.io/en/latest/technical_reference/features.html?highlight=r0912#design-checker-messages

This is a backport from
https://github.com/Electron-Cash/Electron-Cash/pull/2087.

---
This change is [Reviewable](https://reviewable.io/reviews/bitcoin-abc/electrumabc/24)

### CODE REVIEW

**Comment by gasull** [comment]
The diff can be seen better [on Reviewable](https://reviewable.io/reviews/Bitcoin-ABC/ElectrumABC/24#-).

**Comment by PiRK** [comment]
I did a pull request on your pull request to tackle a few more issues in the original code. If you  don't like it, it's fine. Your refactoring is already a clear improvement and I can accept it as it is.

In the long run, I'm wondering if those command line commands are really needed or if we should just drop them.

**Comment by gasull** [comment]
> I did a pull request on your pull request to tackle a few more issues in the original code. If you don't like it, it's fine.

I just wanted to say that the improvements in https://github.com/Bitcoin-ABC/ElectrumABC/pull/35 are great.

> In the long run, I'm wondering if those command line commands are really needed or if we should just drop them.

I'm not sure what you mean with "commands" here.  Do you mean `create_wallet` and `restore_wallet` functions?  I don't see why we would want to drop them.

**Comment by gasull** [comment]
Oh, I see: `./electrum-abc create`.

I guess it depends on the goals.  If one of the goals is having a command-line wallet that can be used by other software, then I wouldn't delete the commands.
